tests/core/test_model.py

```diff
@@ -1373,14 +1373,16 @@ def test_parse(assert_exp_eq):
 
         SELECT
           id::INT AS id,
-          ds
+          ds,
+          foo
         FROM x
         WHERE ds BETWEEN '{{ start_ds }}' AND @end_ds
     """
     )
     model = load_sql_based_model(expressions, dialect="hive")
     assert model.columns_to_types == {
-        "ds": exp.DataType.build("unknown"),
+        "foo": exp.DataType.build("unknown"),
```

Comment:
This test was explicitly checking for the time_column to be `unknown`. Since my change treats unknown time_column's as timestamps, I added another non time_column to this test to show that `unknown` still works for columns that arent the `time_column`